elm/Main.elm

```diff
@@ -85,11 +59,32 @@ update msg model =
                     |> Maybe.fromJust
 
             in
-               { model | setModels = models }
+               ({ model | setModels = models }, Cmd.none)
+
+        MovementInputMsg msg ->
+            let
+                (updated, cmd) = MovementInput.update msg model.movements
+            in
+                ({ model | movements = updated }, Cmd.map MovementInputMsg cmd)
+
+        AddSet ->
+            let
+                count = 0
+                newSets = model.setModels ++ [SingleSet.initialModel count]
```

Comment:
`SingleSet.initialModel 0` is used in a couple of spots in the codebase; is it ever called with a value other than `0`? If not, thoughts on removing that argument?

Reply:
Yes, definitely. That's an artifact from some old design decisions that I'm planning on refactoring/rewriting relatively soon. I intentionally avoided diving into that rabbit hole during these changes.

Reply:
👍